tests/beta/test_default_tensor.py

```diff
@@ -126,8 +126,15 @@ def prep_par(par, op):
     return par
 
 
-class TestAuxillaryFunctions:
-    """Test auxillary functions."""
+def edges_valid(dev, num_nodes):
+    """Test if the edges in a device are properly accounted for, when there are num_nodes in tensor network"""
```

Comment:
Since this isn't an assertion:
```suggestion
    """Returns True if the edges in a device are properly accounted for, when there are num_nodes in tensor network"""
```